x-pack/metricbeat/module/azure/compute_vm_scaleset/manifest.yml

```diff
@@ -12,8 +12,10 @@ input:
       - name: ["CPU Credits Remaining", "CPU Credits Consumed", "OS Per Disk Read Bytes/sec", "OS Per Disk Write Bytes/sec", "OS Per Disk Read Operations/Sec", "OS Per Disk Write Operations/Sec", "OS Per Disk QD"]
         namespace: "Microsoft.Compute/virtualMachineScaleSets"
         timegrain: "PT5M"
+        ignore_unsupported: true
```

Comment:
Should `ignore_unsupported` config parameter be a higher level config so you do have you set it for every individual metrics config? For example:
```
resources:
- resource_group: ""
  resource_type: "Microsoft.Compute/virtualMachineScaleSets"
  ignore_unsupported: true
  metrics:
  - name: ["CPU Credits Remaining", "CPU Credits Consumed"]
     namespace: "Microsoft.Compute/virtualMachineScaleSets"
     timegrain: "PT5M"
```

Reply:
I would leave it at metric/namespace level, in case users want to ignore only the namespaces/metrics that some resources support, as in the case for windows vs linux machines. Adding it at resource level will leave too much room for mistakes as wrong spelling etc.